apps/bazel-is-cool/BUILD.bazel

```diff
@@ -0,0 +1,67 @@
+load("@npm//@angular/cli:index.bzl", "ng", "ng_test")
+
+filegroup(
+    name = "srcs",
+    srcs = glob([
+        "src/*.ts",
+        "src/*.css",
+        "src/*.html",
+        "src/**/*.ts",
+        "src/**/*.css",
+        "src/**/*.html",
+    ]),
+)
+
+ng(
+    name = "build",
+    args = [
+        "build",
+        "bazel-is-cool",
+        "--outputPath=$@",
+    ],
+    data = [
+        "browserslist",
+        "src/favicon.ico",
+        "tsconfig.app.json",
+        "tsconfig.json",
+        ":srcs",
```

Comment:
Doing this will trigger builds whenever you change a spec file

Reply:
Done